Firebase/Core/Public/FIRVersion.h

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2017 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import <Foundation/Foundation.h>
+
+// The version of the Firebase SDK.
+extern NSString *const kFIRLibraryVersionID;
```

Comment:
I see your declaration of kFIRLibraryVersionID here but I don't see a definition. Are you missing a file?

Reply:
No, it turns out it did already exist. It's in FIROptions.m[0]. It is possible that we would like a new home for it though. I'm hoping someone from the core team can give some guidance there.

This constant is also not currently tied to the one actually used to build the sdk by anything other than policy. That being said, it does appear to be current, so that might be good enough for now.

0: https://github.com/firebase/firebase-ios-sdk/blob/9ad65fe00f78302aa025e3e9ff753fbc0989599f/Firebase/Core/FIROptions.m#L42